integration/nwo/configtx_template.go

```diff
@@ -26,6 +27,20 @@ Organizations:{{ range .PeerOrgs }}
     Admins:
       Type: Signature
       Rule: OR('{{.MSPID}}.admin')
+    {{- else }}
+    Readers:
+      Type: Signature
+      Rule: OR('{{.MSPID}}.member')
+    Writers:
+      Type: Signature
+      Rule: OR('{{.MSPID}}.member')
+    Endorsement:
+      Type: Signature
+      Rule: OR('{{.MSPID}}.member')
+    Admins:
+      Type: Signature
+      Rule: OR('{{.MSPID}}.member')
```

Comment:
I don't think we want this.  Even without node-OUs, your admins should still require the admin role.